specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/assessments.json

```diff
@@ -242,7 +242,7 @@
       "description": "Describes properties of an assessment.",
       "properties": {
         "resourceDetails": {
-          "$ref": "../../../common/v1/types.json#/definitions/ResourceDetails"
```

Comment:
I deleted the path ".. / .. / .. / common / v1 / types.json" because I updated the types.json file in a new version of ResourceDetails object, so to maintain compatibility I moved the ResourceDetails definition as defined in the types.json file before my changes